presto-hive/src/test/java/io/prestosql/plugin/hive/orc/TestOrcAcidPageSource.java

```diff
@@ -11,20 +11,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.prestosql.plugin.hive.acid;
+package io.prestosql.plugin.hive.orc;
```

Comment:
this change belongs to `Add support for reading ORC ACID files`

Reply:
That change doesnt support delete deltas hence changes are done in this commit